src/core/imap/MCIMAPSession.cpp

```diff
@@ -2813,8 +2813,9 @@ static void nstringDeallocator(char * bytes, unsigned int length) {
     mailimap_nstring_free(bytes);
 };
 
-Data * IMAPSession::fetchMessageAttachment(String * folder, bool identifier_is_uid,
+Data * IMAPSession::fetchNonDecodedMessageAttachment(String * folder, bool identifier_is_uid,
                                            uint32_t identifier, String * partID,
+                                           uint32_t offset, uint32_t length,
                                            Encoding encoding, IMAPProgressCallback * progressCallback, ErrorCode * pError)
```

Comment:
remove encoding from parameters of fetchNonDecodedMessageAttachment() if it's not used.